quipucords/scanner/network/connect_callback.py

```diff
@@ -52,6 +54,7 @@ def v2_runner_on_ok(self, result):
 
     def v2_runner_on_unreachable(self, result):
         """Print a json representation of the result."""
+        input_log.log_ansible_result(result)
```

Comment:
May want to try/catch around this so any IO or other related failure in this flow doesn't kill a scan.

Reply:
Done.